### DISCUSSION TITLE
How can I get records with true only?

### DISCUSSION THREAD

**Discussion by thanksyouall** [carltongibson/django-filter#1371] [Q&A] [answered]
I want to get records whose is_archive field is always true. How can I do this?

I tried:
`is_all_archive = BooleanFilter(field_name='is_archive`)

But then I can filter by is_archive=False. 
I tried adding lookup_expr='istrue', but it doesn't work.

**Comment by thanksyouall** [comment]
@carltongibson can you help me please?

**Comment by saadmk11** [comment, marked as answer]
If you want to filter only the records which have `is_archive=True` you can override the views `get_queryset()` method.
for example:

```python
def get_queryset(self):
    return super().get_queryset().filter(is_archive=True)
```

Or you can override the `filter_queryset()` method of your filterset class.
for example:

```python
def filter_queryset(self, queryset):
    return super().filter_queryset(queryset).filter(is_archive=True)
```

**Reply by thanksyouall** [reply]
Thanks!
